lib/v2/eastmoney/report/index.js

```diff
@@ -0,0 +1,53 @@
+const cheerio = require('cheerio');
```

Comment:
No need to use puppeteer at all. The data is embedded as a JSON `initdata` in HTML.

Reply:
ok